cmd/kube-apiserver/app/aggregator.go

```diff
@@ -269,6 +269,7 @@ var apiVersionPriorities = map[schema.GroupVersion]priority{
 	{Group: "scheduling.k8s.io", Version: "v1alpha1"}:            {group: 16600, version: 9},
 	{Group: "coordination.k8s.io", Version: "v1"}:                {group: 16500, version: 15},
 	{Group: "coordination.k8s.io", Version: "v1beta1"}:           {group: 16500, version: 9},
+	{Group: "node.k8s.io", Version: "v1"}:                        {group: 16300, version: 15},
```

Comment:
I wasn't sure about the version number. `15` comes from other API examples in this file